tests/paz/pipelines/pix2pose_metric_test.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+import cv2
+import os
+from tensorflow.keras.utils import get_file
+from paz.backend.image import load_image
+from paz.backend.camera import Camera
+from paz.pipelines import PIX2POSEPowerDrill
+from paz.backend.groups.quaternion import quaternion_to_rotation_matrix, rotation_matrix_to_quaternion
+import re
+from scipy import spatial
+from scipy.spatial import distance
+import numpy as np
+from paz.abstract.messages import Pose6D
+from examples.pix2pose.eval import compute_ADI, compute_ADD, transform_mesh_points, get_translations, get_quaternion
+
+
+@pytest.fixture
+def image_PowerDrill():
+    URL = ('https://github.com/oarriaga/altamira-data/releases/download'
+           '/v0.9.1/image_with_YCB_objects.jpg')
+    filename = os.path.basename(URL)
+    fullpath = get_file(filename, URL, cache_subdir='paz/tests')
+    image = load_image(fullpath)
+    return image
+
+
+@pytest.fixture
+def boxes_Pix2Pose():
+    rotation = np.array(
+        [-0.9078519116941933, -0.4175345487208392, 0.038330959211728285, 0.3050982069800776, -0.5951250307633585,
+         0.743466715795677, -0.28761117420744436, 0.6866522290644125, 0.6676740178175343]).reshape(3, 3)
+    translation = np.array([104.54312464515644, 71.49346046440272, 955.3250389603148]).reshape(3, 1)
+    return [rotation, translation]
+
+
+def assert_get_quaternion_translation(pred_box):
```

Comment:
please change this name to have the name test at the end. For example the name of this function could be something like: "quaternion_translation_shape_test". The "test" at the end is relevant since pytest searches for all functions with test in their name.

Reply:
Also apply these for all the tests, the fixtures don't have to have that since fixture are just the "inputs".

Reply:
Resolved